sdk/formrecognizer/azure-ai-formrecognizer/tests/test_custom_forms.py

```diff
@@ -142,7 +215,45 @@ def callback(raw_response, _, headers):
         self.assertEqual(recognized_form[0].page_range.last_page, page_results[0].page)
         self.assertUnlabeledFormFieldDictTransformCorrect(recognized_form[0].fields, actual_fields, read_results)
 
-    @GlobalFormAndStorageAccountPreparer()
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalTrainingAccountPreparer(multipage=True)
+    def test_custmform_multipg_unlbld_trnsfrm(self, client, container_sas_url):
```

Comment:
I'm willing to bet it had something to do with using storage and their min char requirements because I definitely needed to make the name shorter. I'll see if I can update some of the names or if this is still a constraint for cog services.

Reply:
Yep definitely a constraint of storage. I can use real english words now!! :)